### PR TITLE
Add support for 'Export Several Images'

### PR DESCRIPTION
**Pull request by bjfr**
https://docs.docker.com/engine/api/v1.31/#operation/ImageGetAll

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/1018)

### CODE REVIEW

File: src/main/java/com/github/dockerjava/core/AbstractDockerCmdExecFactory.java [hunk 1]
```diff
@@ -197,6 +198,11 @@ public void init(DockerClientConfig dockerClientConfig) {
         return new SaveImageCmdExec(getBaseResource(), getDockerClientConfig());
     }
 
+    @Override
+    public SaveImagesCmd.Exec createSaveImagesCmdExec() {
+        return null;
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/core/AbstractDockerCmdExecFactory.java]: why null?

**Comment by bjfr** [comment]
#1019

File: src/main/java/com/github/dockerjava/api/command/SaveImagesCmd.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import com.github.dockerjava.api.exception.NotFoundException;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import java.io.InputStream;
+import java.util.List;
+
+public interface SaveImagesCmd extends SyncDockerCmd<InputStream> {
+
+    //CHECKSTYLE:OFF
```
[the remark this answers is not in the document]
Comment (bjfr) [reply src/main/java/com/github/dockerjava/api/command/SaveImagesCmd.java]: It is not possible to save more than one image in a tar using the existing SaveImageCmd. But the docker API has that possibility.

[the remark this answers is not in the document]
Comment (KostyaSha) [reply src/main/java/com/github/dockerjava/api/command/SaveImagesCmd.java]: I mean checkstyle, why do you need disable it?

**Pull request closed without merging**